Sources/iosDesignTokens/todaytix/Tokens.swift

```diff
@@ -0,0 +1,282 @@
+    public static let darkTagBrandHover = UIColor(red: 0.000, green: 0.000, blue: 0.000, alpha: 1)
+    public static let darkTagBrandText = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let darkTagGrayBackground = UIColor(red: 0.753, green: 0.753, blue: 0.776, alpha: 1)
+    public static let darkTagGrayHover = UIColor(red: 0.671, green: 0.671, blue: 0.698, alpha: 1)
+    public static let darkTagGrayText = UIColor(red: 0.043, green: 0.043, blue: 0.063, alpha: 1)
+    public static let darkTagGreenBackground = UIColor(red: 0.455, green: 0.878, blue: 0.573, alpha: 1)
+    public static let darkTagGreenHover = UIColor(red: 0.271, green: 0.835, blue: 0.427, alpha: 1)
+    public static let darkTagGreenText = UIColor(red: 0.043, green: 0.043, blue: 0.063, alpha: 1)
+    public static let darkTagOrangeBackground = UIColor(red: 1.000, green: 0.698, blue: 0.341, alpha: 1)
+    public static let darkTagOrangeHover = UIColor(red: 0.980, green: 0.620, blue: 0.200, alpha: 1)
+    public static let darkTagOrangeText = UIColor(red: 0.043, green: 0.043, blue: 0.063, alpha: 1)
+    public static let darkTagPurpleBackground = UIColor(red: 0.671, green: 0.380, blue: 0.961, alpha: 1)
+    public static let darkTagPurpleHover = UIColor(red: 0.635, green: 0.337, blue: 0.929, alpha: 1)
+    public static let darkTagPurpleText = UIColor(red: 0.043, green: 0.043, blue: 0.063, alpha: 1)
+    public static let darkTagRedBackground = UIColor(red: 1.000, green: 0.435, blue: 0.604, alpha: 1)
+    public static let darkTagRedHover = UIColor(red: 1.000, green: 0.247, blue: 0.475, alpha: 1)
+    public static let darkTagRedText = UIColor(red: 0.043, green: 0.043, blue: 0.063, alpha: 1)
+    public static let darkText01 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let darkText02 = UIColor(red: 0.835, green: 0.835, blue: 0.851, alpha: 1)
+    public static let darkText03 = UIColor(red: 0.671, green: 0.671, blue: 0.698, alpha: 1)
+    public static let darkText04 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let darkText05 = UIColor(red: 0.588, green: 0.588, blue: 0.624, alpha: 1)
+    public static let darkTextDiscount = UIColor(red: 1.000, green: 0.059, blue: 0.341, alpha: 1)
+    public static let darkTextError = UIColor(red: 1.000, green: 0.071, blue: 0.149, alpha: 1)
+    public static let darkUiUi01 = UIColor(red: 0.090, green: 0.086, blue: 0.122, alpha: 1)
+    public static let darkUiUi02 = UIColor(red: 0.173, green: 0.173, blue: 0.247, alpha: 1)
+    public static let darkUiUi03 = UIColor(red: 0.392, green: 0.388, blue: 0.455, alpha: 1)
+    public static let darkUiUi04 = UIColor(red: 0.588, green: 0.588, blue: 0.624, alpha: 1)
+    public static let darkUiUi05 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let fontFamiliesBody = "Aktiv Grotesk App"
```

Comment:
Do we need to add the fonts as well or just define the names?

Reply:
just defining the names for now - still sort of deciding how to do typography. This is all still a work in progress